ran-core/src/main/java/io/ran/CrudRepoBaseQuery.java

```diff
@@ -13,7 +13,7 @@
 	protected Class<T> clazz;
 	protected TypeDescriber<T> typeDescriber;
 	protected GenericFactory genericFactory;
-	protected Property currentProperty = Property.get(Token.of("t"), Clazz.of(Object.class));
+	protected Property currentProperty = null;
```

Comment:
What is this field used for, it is not used by the class itself, since it's protected it could be used by subclasses... but it's not clear for what. Maybe a javacomment to explain, or remove it if it is not actually being used?